plutus-core/plutus-core/test/TypeSynthesis/Golden/Signatures/DefaultFun/ExpModInteger.sig.golden

```diff
@@ -1 +1 @@
-Integer -> Integer -> Natural -> BuiltinResult Natural
\ No newline at end of file
+IntegerCostedByNumBytes -> IntegerCostedByNumBytes -> IntegerCostedByNumBytes -> BuiltinResult Natural
```

Comment:
How are these golden files generated? I don't see a type called `IntegerCostedByNumBytes` in the codebase?

Reply:
Oh, I removed that.  I updated most of the golden files but must have forgotten this one.  I'll fix that.